### PR TITLE
[MarginPage] update data-action-ids

### PR DESCRIPTION
**Pull request by ShynRou**
Resolves https://sovryn.monday.com/boards/2218344956/pulses/2592953180

### CODE REVIEW

**Review by tiltom — DISMISSED** [review]

**Review by creed-victor — DISMISSED** [review]

**Review by soulBit — APPROVED** [review]
I've applied one small fix for missing action-id's on the open position table, and updated the spec to be consistent with the implementation - changes look good and can be merged once QA approve 👍🏻

**Review by Alitasovryn — APPROVED** [review]

**Pull request merged**
